docs/4-At-The-Internship/asking_questions.md

```diff
@@ -24,6 +26,7 @@ This is a trick question. I don't think there's such thing as too many questions
 - **Could you have used standard debugging/code searching tricks to figure it out?** If you don't know these yet, then ask someone to help you. But notice what stategies they use to figure it out. Next time you have a problem, those same strategies might help.
 - **How long will it take for them to answer your question?** This is often unknown. But try to think of how much questions will bother other ppl vs benefit you. Some quick questions are a small amount of time on their part and a huge benefit to your comfort and ability.
 
+Lunch time tends to be a great time to ask questions, since people aren't focused on their work. Try to Never Eat Alone - there's even a book by that name!
```

Comment:
this isn't really answering the "too much"/"what's a good question" question, maybe add a small section for when to ask? and add stuff about being considerate when people are in the middle of working

Also I don't think we should recommend people never eat alone - some people really enjoy the quiet time of a meal by themselves and I don't think there's anything wrong with that. Again, maybe say instead 

> Lunch time tends to be a great time to ask questions, since people aren't focused on their work. Rudi recommends the book "Never Eat Alone".